doc/Output JSON.json

```diff
@@ -263,7 +292,7 @@
           "text": "Pregnancy status"
         },
         "entry": [ {
-          "reference": "Observation/5e085278-4d3b-41e7-9d26-d8ae0dab6a84"
+          "reference": "Observation/30ee6d70-da6c-4db9-8954-461a7d4cbbb4"
         } ]
       }, {
         "title": "Mental Health Assesment",
```

Comment:
again .. no separate section. See above comment for using observation for mental health assessment.